com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,128 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: "43"
```

Comment:
Your stuff is buggy, or your instructions are wrong.

How do I get 44?  I type flatpak update and it says Nothing to do.

It's really frustrating that your instructions are vague and incorrect.

Reply:
How do I update to 44?  I'm on Debian, it doesn't work.

Reply:
```
$ flatpak update org.gnome.Platform
Looking for updates…
Nothing to do.

 $ flatpak update org.gnome.Sdk
Looking for updates…
Nothing to do.

$ flatpak update
Looking for updates…
Nothing to do.

$ flatpak list
Name                                               Application ID                                         Version           Branch               Origin                Installation
Deckr                                              com.cocoatech.deckr                                                      master               deckr-origin          user
Freedesktop Platform                               org.freedesktop.Platform                               21.08.18          21.08                flathub               system
Freedesktop Platform                               org.freedesktop.Platform                               22.08.9           22.08                flathub               system
Mesa                                               org.freedesktop.Platform.GL.default                    21.3.9            21.08                flathub               system
Mesa                                               org.freedesktop.Platform.GL.default                    22.3.5            22.08                flathub               system
Mesa (Extra)                                       org.freedesktop.Platform.GL.default                    22.3.5            22.08-extra          flathub               system
nvidia-470-141-03                                  org.freedesktop.Platform.GL.nvidia-470-141-03                            1.4                  flathub               system
nvidia-470-161-03                                  org.freedesktop.Platform.GL.nvidia-470-161-03                            1.4                  flathub               system
openh264                                           org.freedesktop.Platform.openh264                      2.1.0             2.0                  flathub               system
openh264                                           org.freedesktop.Platform.openh264                      2.1.0             2.2.0                flathub               system
Freedesktop SDK                                    org.freedesktop.Sdk                                    21.08.18          21.08                flathub               system
Freedesktop SDK                                    org.freedesktop.Sdk                                    22.08.9           22.08                flathub               system
GNOME Application Platform version 43              org.gnome.Platform                                                       43                   flathub               system
GNOME Software Development Kit version 43          org.gnome.Sdk                                                            43                   flathub               system
Adwaita dark GTK theme                             org.gtk.Gtk3theme.Adwaita-dark                                           3.22                 flathub               system
```

Reply:
How about `flatpak install org.gnome.Sdk` ? It will be nice and even offer different versions. 

It's in the documentation too

https://docs.flatpak.org/en/latest/first-build.html#install-a-runtime-and-the-matching-sdk

Reply:
You'll need to install the Platform.